nana/modules/youtube.py

```diff
@@ -61,21 +60,21 @@ async def youtube_download(_client, message):
 		stream.download('nana/downloads', filename="tempvid")
 		await app.send_video(message.chat.id, video="nana/downloads/tempvid.mp4")
 		os.remove('nana/downloads/tempvid.mp4')
-		await message.reply(text, disable_web_page_preview=True)
+		await edrep(message, text=text, disable_web_page_preview=True)
 		return
 
 
-@app.on_message(filters.user(AdminSettings) & filters.command(["ytmusic", "ytaudio"], Command))
+@app.on_message(filters.user(AdminSettings) & filters.command("ytmusic", Command))
```

Comment:
Codacy found an issue: [youtube_music is too complex (27) (MC0001)](https://app.codacy.com/gh/MadeByThePinsHub/Nana-Remix/pullRequest?prid=6330176)